packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.tsx

```diff
@@ -18,10 +18,11 @@ export class DatacomBreadcrumb {
     return (
       <Host>
         {!!this.url ? (
-          <div>
-            <a href={this.url} class="dc-breadcrumb">
-              <slot></slot> {this.NavIcon}
+          <div class="dc-breadcrumb">
+            <a href={this.url} class=" dc-text">
```

Comment:
You got an empty space before your class here. Please remove for code cleanliness.

Reply:
Done